### PR TITLE
6 add registration authorisation

### PR DESCRIPTION
**Pull request by homomorfism**
Closing #6 . Added registration, authorization and added main page.

### CODE REVIEW

File: biproductive/templates/static/scripts/game.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981738/memory/lotr/bilbo.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981802/memory/lotr/frodo.jpg'
+    ],
+    disney: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982044/memory/disney/mickey.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982088/memory/disney/mowgli.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982610/memory/disney/tarzan.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982620/memory/disney/simba.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982628/memory/disney/aladin.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982636/memory/disney/blanche%20neige.jpg',
```
Comment (rizvansky) [inline biproductive/templates/static/scripts/game.js:77]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547982636/memory/disney/blanche%20neige.jpg",
```

File: biproductive/templates/static/scripts/game.js [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981054/memory/starwars/Han%20solo.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981074/memory/starwars/chewbacca.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981095/memory/starwars/yoda.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981117/memory/starwars/dark%20sidious.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981141/memory/starwars/dark%20vador.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981165/memory/starwars/padme.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981190/memory/starwars/leia.jpg'
+    ],
+    lotr: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981408/memory/lotr/gandalf.jpg',
```
Comment (rizvansky) [inline biproductive/templates/static/scripts/game.js:50]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547981408/memory/lotr/gandalf.jpg",
```

File: biproductive/templates/static/scripts/game.js [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981501/memory/lotr/legolas.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981535/memory/lotr/Gimli.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981603/memory/lotr/golum.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981645/memory/lotr/sam.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981686/memory/lotr/saroumane.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981738/memory/lotr/bilbo.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981802/memory/lotr/frodo.jpg'
+    ],
+    disney: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547982044/memory/disney/mickey.jpg',
```
Comment (rizvansky) [inline biproductive/templates/static/scripts/game.js:72]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547982044/memory/disney/mickey.jpg",
```

File: biproductive/templates/static/scripts/game.js [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981095/memory/starwars/yoda.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981117/memory/starwars/dark%20sidious.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981141/memory/starwars/dark%20vador.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981165/memory/starwars/padme.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981190/memory/starwars/leia.jpg'
+    ],
+    lotr: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981408/memory/lotr/gandalf.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981438/memory/lotr/sauron.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981469/memory/lotr/Aragorn.jpg',
```
Comment (rizvansky) [inline biproductive/templates/static/scripts/game.js:52]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547981469/memory/lotr/Aragorn.jpg",
```

File: biproductive/templates/static/scripts/game.js [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980025/memory/Pokemon/Bulbasaur.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980083/memory/Pokemon/Charmander.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980101/memory/Pokemon/Squirtle.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980116/memory/Pokemon/Pikachu.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980129/memory/Pokemon/Mewtwo.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980142/memory/Pokemon/Mew.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980154/memory/Pokemon/Articuno.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980164/memory/Pokemon/Zapdos.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980175/memory/Pokemon/Moltres.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980186/memory/Pokemon/Eevee.png',
```
Comment (rizvansky) [inline biproductive/templates/static/scripts/game.js:15]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547980186/memory/Pokemon/Eevee.png",
```

File: biproductive/templates/static/scripts/game.js [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980175/memory/Pokemon/Moltres.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980186/memory/Pokemon/Eevee.png'
+    ],
+    starwars: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980981/memory/starwars/anakin%20skywalker.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981009/memory/starwars/luke%20skywalker.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981022/memory/starwars/Obi%20wann.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981054/memory/starwars/Han%20solo.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981074/memory/starwars/chewbacca.jpg',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547981095/memory/starwars/yoda.jpg',
```
Comment (rizvansky) [inline biproductive/templates/static/scripts/game.js:33]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547981095/memory/starwars/yoda.jpg",
```

File: biproductive/templates/static/scripts/game.js [hunk 7]
```diff
@@ -0,0 +1,298 @@
+// other themes to add ?
+// bigger memory?
+
+const library = {
+    pokemon: [
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980025/memory/Pokemon/Bulbasaur.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980083/memory/Pokemon/Charmander.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980101/memory/Pokemon/Squirtle.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980116/memory/Pokemon/Pikachu.png',
+        'https://res.cloudinary.com/beumsk/image/upload/v1547980129/memory/Pokemon/Mewtwo.png',
```
Comment (rizvansky) [inline biproductive/templates/static/scripts/game.js:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rizvansky/biproductive/pullRequest?prid=8150207)
```suggestion
        "https://res.cloudinary.com/beumsk/image/upload/v1547980129/memory/Pokemon/Mewtwo.png",
```

**Review by implausibleDenyability — APPROVED** [review]

**Pull request merged**